docs/openstack-cloud-controller-manager/expose-applications-using-loadbalancer-type-service.md

```diff
@@ -130,49 +130,69 @@ Request Body:
 
   If 'true', the loadbalancer pool protocol will be set as `PROXY`. Default is 'false'.
 
+  NOTE: Not supported with `ovn` provider.
+
 - `loadbalancer.openstack.org/x-forwarded-for`
 
   If 'true', `X-Forwarded-For` is inserted into the HTTP headers which contains the original client IP address so that the backend HTTP service is able to get the real source IP of the request. Please note that the cloud provider will force the creation of an Octavia listener of type `HTTP` if this option is set. Only applies when using Octavia.
 
   This annotation also works in conjunction with the `loadbalancer.openstack.org/default-tls-container-ref` annotation. In this case the cloud provider will create an Octavia listener of type `TERMINATED_HTTPS` instead of an `HTTP` listener.
 
+  NOTE: Not supported with `ovn` provider.
+
 - `loadbalancer.openstack.org/timeout-client-data`
 
   Frontend client inactivity timeout in milliseconds for the load balancer.
 
+  NOTE: Not supported with `ovn` provider.
+
 - `loadbalancer.openstack.org/timeout-member-connect`
 
   Backend member connection timeout in milliseconds for the load balancer.
 
+  NOTE: Not supported with `ovn` provider.
+
 - `loadbalancer.openstack.org/timeout-member-data`
 
   Backend member inactivity timeout in milliseconds for the load balancer.
 
+  NOTE: Not supported with `ovn` provider.
+
 - `loadbalancer.openstack.org/timeout-tcp-inspect`
 
   Time to wait for additional TCP packets for content inspection in milliseconds for the load balancer.
 
+  NOTE: Not supported with `ovn` provider.
```

Comment:
`NOTE` is not needed here.

My suggestion: 
```
Not supported when `lb-provider=ovn` is configured in openstack-cloud-controller-manager.
```

Reply:
Done